app/js/controllers/main.js

```diff
@@ -70,6 +77,22 @@ export default class MainController extends BaseController {
   }
 
   /**
+   * Clear all data/settings stored on the browser. Use with caution.
+   *
+   * @param {boolean} ignoreServiceWorker
+   * @return {Promise}
+   */
+  clear(ignoreServiceWorker = true) {
+    const promises = [this[p.settings].clear()];
+
+    if (!ignoreServiceWorker) {
+      promises.push(this[p.server].clearServiceWorker());
```

Comment:
as you suggested, I moved the service worker clear code back to `server/index.js`.

But because we instantiate the settings object here, I think we need to clear it here as well. We can revisit once we switch to react/router (which I don't know TBH :) )